sdcm/cluster.py

```diff
@@ -3211,9 +3287,11 @@ def get_datacenter_name_per_region(self, db_nodes=None):
         datacenter_name_per_region = {}
         for region, nodes in self.nodes_by_region(nodes=db_nodes).items():
             if status := nodes[0].get_nodes_status():
-                datacenter_name_per_region[region] = status[nodes[0]]['dc']
+                # If `nodetool status` failed to get status for the node
+                if dc_name := status.get(nodes[0]).get('dc'):
```

Comment:
To make it be really safe the first `get` method should have default value defined:
```
if dc_name := status.get(nodes[0], {}).get('dc'):
```

Reply:
Where?

Reply:
forgot to push ..
pushed